prettier/tests/run.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: prettier-pipeline
+spec:
+  workspaces:
+  - name: shared-workspace
+    # description: The git repo will be cloned into this workspace and the config-defaults.yaml will be read from it.
+  tasks:
+  - name: fetch-repo
+    taskRef:
+      name: git-clone
+    workspaces:
+    - name: output
+      workspace: shared-workspace
+    params:
+    - name: url
+      value: https://github.com/tektoncd/pipeline.git
+    - name: revision
+      value: 6b73bb776a0a32fcaddc59ce4aa6c149f24df945
+  - name: check-or-format
+    taskRef:
+      name: prettier
+    runAfter:
+    - fetch-repo
+    workspaces:
+    - name: source
+      workspace: shared-workspace
+    params: 
+    - name: args
+      value: 
+        - --write 
+        - "config/config-defaults.yaml"
```

Comment:
curious, is this default config coming from the container image? and users are able to specify a prettier config file from the workspace right?

Reply:
This is the file from the git repo used by `prettier` to format. It acts as a test file to be formatted in the repository.

Reply:
ahh! ignore my comment! I was thinking of the [prettier config file](https://prettier.io/docs/en/configuration.html) which allows the user to set their linter settings...but it looks like prettier would auto-detect the file based on file naming convention.